dashboard/test/ui/features/teacher_tools/pl_landing_page.feature

```diff
@@ -0,0 +1,123 @@
+    And I wait until element "button:contains(Instructor Center)" is visible
+    Then I click selector "button:contains(Instructor Center)"
+
+    # Sees Instructor Professional Learning Sections section
+    And I wait until element "button:contains(Create a section)" is visible
+
+  Scenario: Regional Partner sees relevant content sections
+    Given I am a program manager with started and completed courses
+    And I am on "http://studio.code.org/my-professional-learning"
+
+    # Go to the right My PL page tab
+    And I wait until element "button:contains(Regional Partner)" is visible
+    Then I click selector "button:contains(Regional Partner)"
+
+    # Sees Regional Partner Resources section
+    And I wait until element "a:contains(Manage applications)" is visible
+    And the href of selector "a:contains(Manage applications)" contains "/pd/application_dashboard"
+    And element "a:contains(View workshop dashboard)" is visible
+    And the href of selector "a:contains(View workshop dashboard)" contains "/pd/workshop_dashboard"
+    And element "a:contains(View playbook)" is visible
+    And the href of selector "a:contains(View playbook)" contains "/educate/regional-partner/playbook"
+
+    # Sees Workshops table
+    And I wait until element "button:contains(Workshop Details)" is visible
+
+  Scenario: Workshop Organizer sees relevant content sections
+    Given I am an organizer with started and completed courses
+    And I am on "http://studio.code.org/my-professional-learning"
+
+    # Go to the right My PL page tab
+    And I wait until element "button:contains(Workshop Organizer)" is visible
+    Then I click selector "button:contains(Workshop Organizer)"
+
+    # Sees Workshop Organizer Resources section
+    And I wait until element "a:contains(View workshop dashboard)" is visible
+    And the href of selector "a:contains(View workshop dashboard)" contains "/pd/workshop_dashboard"
+
+    # Sees Workshops table
+    And I wait until element "button:contains(Workshop Details)" is visible
+
+  Scenario: Teacher with Self-paced PL courses sees relevant content sections
+    Given I create a teacher named "Self-paced Teacher"
+    And I sign in as "Self-paced Teacher" and go home
+    Then I am on "http://studio.code.org/my-professional-learning"
+
+    # Sees Joined Professional Learning Sections section
+    And element "button.ui-test-join-section" is visible
+
+    # Sees Recommended for you section
+    And element "a:contains(Learn more about workshops)" is visible
+    And the href of selector "a:contains(Learn more about workshops)" contains "/professional-learning/middle-high"
+    And element "a:contains(Start professional learning courses)" is visible
+    And the href of selector "a:contains(Start professional learning courses)" contains "/educate/professional-development-online"
+
+    # Starts a self-paced PL course
+    Then I am on "http://studio.code.org/s/self-paced-pl-csd-unit2-1-2024"
```

Comment:
This script isn't seeded on our test servers. Why didn't `alltheselfpacedthings` work? We can either add something to that script or, if needed, add this script to our test servers to fix the failing test

Reply:
It looked more complicated to use since step 1 required coding to make progress. But I just checked to see if we can skip ahead in the levels and we can (🎉) so I tried going to step 3 and 4 and think that might work. It passed SauceLabs so 🤞 for drone.

Reply:
gotcha! Yeah skipping ahead makes sense here! 🤞

Reply:
It passed! Do the two Eyes tests have anything to do with it saying "4 passed" since there are 6 tests total? 

<img width="644" alt="Screenshot 2024-05-24 at 9 19 37 AM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/12d39c04-6ef6-4326-bb2c-6386a2b9669e">

Reply:
yeah, eyes and non-eyes run separately. So looks like everything passed!!